packages/config/src/path.ts

```diff
@@ -1,5 +1,11 @@
 import path from 'path';
 import module from 'module';
+import chalk from 'chalk';
+
+export enum PackageAliasType {
```

Comment:
Let's not use enums, please :)
I don't like to use enums in public libraries, because TS transforms them into objects with numeric keys and external users can stick on these numbers in their code. It may break their code when we will change the enum items

Reply:
Current implementation translates to:
```js
var PackageAliasType;
(function (PackageAliasType) {
    PackageAliasType["PLUGIN"] = "PLUGIN";
    PackageAliasType["REPORTER"] = "REPORTER";
})(PackageAliasType || (PackageAliasType = {}));
```

Please note that there are no number keys since string enum values were provided in `enum` translation.

If that's not safe enough we could use `const enum` construct which removes `enum` definition from the code [completely](https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgBQIbANYYOZwCCANgJYYQAqAngA5wwDeAUDOgDICqA4gJIByMALwwA5Gi59+ogDSsYAJQCiaAPILKShcLHK1GrbOYBfZs1CRYGHZhz4iZCjXoA6CTwEBuZkA). A [little more](https://twitter.com/mgechev/status/1222538039055941632?s=20) on the subject.

Reply:
I guess that another future proof approach would be not to remove any of enums previously defined, as we have to do in protobuf. That way numbers (set by TS when no string value is provided) would only grow.

Reply:
Its ok if string keys have not changed. No need in const